### ISSUE TITLE
The correct connection

### ISSUE THREAD

**Issue by Lukek21** [thomase1234/esphome-fake-xemex-csmb#11]
Hello Thomas,

I'm trying to copy your solution.
However, I am not an expert. 
Can you give me more information about the correct connection of the components?

I have a WROOM32 and a TTL to RS485 adapter (MAX485).
I think I have to wire like this:

DI --> GPIO18
DE --> GPIO18
RE --> GPIO19
RO --> GPIO19

Correct?
And in addition, I have to take out the hashtag in line 90 and 91 in the Yaml Config, right?

Thanks a lot for your help!
Luk

**Comment by thomase1234** [comment]
Hi Luk,

Thanks for reaching out.

My component didn't require the RE and DE port. Your MAX485 requires them. But please use free GPIO's on your WROOM32 dev board. Don't use the GPIO18 and GPIO19 for RE and DE. Any other unused 2 pins can be used.

And yes, you'll have to remove the hashes and change to numbers to the correct pins.

Best regards,
Thomas

**Comment by Lukek21** [comment]
Hi Thomas,

thank you for your answer!
I have adjusted the wiring and the yaml code. Unfortunately, the wallbox does not change the charging power when I change CT1, CT2 and CT3. I will test it further.

Can the Modbus interface of the wallbox also be disabled?

**Comment by thomase1234** [comment]
> Can the Modbus interface of the wallbox also be disabled?
Yes, DPM can be disabled. You can check if the Wallbox is configured to query the DPM(CSMB) module by connecting the A and B wires properly and checking if the EVSE LED on the DPM module is ON. If it's off, I suggest to raise a case with Shell. They can enable DPM remotely. Once that has been confirmed, disconnect the DPM and use this fake CSMB as Modbus server.
